src/components/atoms/charts/Calendar3D/index.jsx

```diff
@@ -0,0 +1,377 @@
+      __html: `<div class="ic-stats-block ic-stats-top">\n
+    <span class="ic-stats-table">\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">1 year total\n
+    <span class="ic-stats-count">${countTotal}</span>\n
+    <span class="ic-stats-average">${averageCount}</span> per day\n
+    </span>\n
+    <span class="ic-stats-meta ic-stats-total-meta">\n
+    <span class="ic-stats-unit">contributions</span>\n
+    <span class="ic-stats-date">${datesTotal}</span>\n
+    </span>\n
+    </span>\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Busiest day\n
+    <span class="ic-stats-count">${maxCount}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">contributions</span>\n
+    <span class="ic-stats-date">${dateBest}</span>\n
+    </span>\n
+    </span>\n
+    </span>\n
+    </span>\n
+    </div>`,
+    };
+  }
+
+  renderBottomStats() {
+    let streakLongest, datesLongest, streakCurrent, datesCurrent, contribData;
+
+    if (this.props.year) {
+      contribData = this.props.platformData.statistic.years.find(
+        (element) => element.year === this.props.year
+      );
+    } else {
+      contribData = this.props.platformData.statistic.current;
+    }
+
+    let contributionCalendar = contribData.calendar;
+
+    if (contribData.streak.longest) {
+      streakLongest = contribData.streak.longest.totalDays;
+      datesLongest =
+        moment(contribData.streak.longest.startDate).format("MMM DD, YYYY") +
+        " - " +
+        moment(contribData.streak.longest.endDate).format("MMM DD, YYYY");
+    } else {
+      streakLongest = "0";
+      datesLongest = "-";
+    }
+    if (contribData.streak.current.id !== -1) {
+      streakCurrent = contribData.streak.current.totalDays;
+      datesCurrent =
+        moment(contribData.streak.current.startDate).format("MMM DD, YYYY") +
+        " - " +
+        moment(contribData.streak.current.endDate).format("MMM DD, YYYY");
+    } else {
+      streakCurrent = "0";
+      datesCurrent = "-";
+    }
+
+    return {
+      __html: `<div class="ic-stats-block ic-stats-bottom">\n
+    <span class="ic-stats-table">\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Longest streak\n
+    <span class="ic-stats-count">${streakLongest}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">days</span>\n
+    <span class="ic-stats-date">${datesLongest}</span>\n
+    </span>\n
+    </span>\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Current streak\n
+    <span class="ic-stats-count">${streakCurrent}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">days</span>\n
+    <span class="ic-stats-date">${datesCurrent}</span>\n
+    </span>\n
+    </span>\n
+    </span>\n
+    </div>`,
+    };
+  }
+
+  renderIsometrics = () => {
+    if (this.context) {
+      const obelisk = require("obelisk.js");
```

Comment:
Can this be moved to the imports?

Reply:
> Can this be moved to the imports?

@Aichnerc

Reply:
Should be possible. I will move it so that we have all imports in the import region.